package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "cli": "ts-node --require ./app/test/globals.ts --require ./app/src/cli/dev-commands-global.ts app/src/cli/main.ts",
     "test:integration": "cross-env TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe --auto-servernum -- mocha -t 30000 --require ts-node/register app/test/integration/*.ts",
-    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 node_modules/.bin/electron ./node_modules/jest/bin/jest --silent --config ./jest.config.js",
+    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 node_modules/.bin/electron ./node_modules/jest/bin/jest --detectOpenHandles --silent --config ./jest.config.js",
```

Comment:
you probably don't want this flag on at all times - the overhead of collecting everything async is pretty atrocious (30%+ overhead on node 8)

Reply:
thanks @SimenB! that's good context to have.

it seems to me like the tradeoff is speed vs possible false positives and i'm not sure that speed is worth the risk of missing failing tests. but perhaps there's another way?

Reply:
If you can put together a test case showing that jest exits with `0` when a test times out, we can fix it 🙂 That's a bug, you shouldn't get false positives

Reply:
💖

Reply:
We do have an open PR for when a test synchronously takes a long time without timing out (which we currently only detect if it happens async), might be it? https://github.com/facebook/jest/pull/6950